configs/panoptic_fpn/panoptic_fpn_r2_50_fpn_fp16_1x_coco.py

```diff
@@ -0,0 +1,12 @@
+_base_ = '../panoptic_fpn/panoptic_fpn_r50_fpn_1x_coco.py'
```

Comment:
Seems this is a new config for panoptic_fpn. Please add the performance in panoptic_fpn README.md